.github/workflows/release.yaml

```diff
@@ -0,0 +1,41 @@
+name: "Create draft release with binaries after tag"
+on:
+  push:
+    tags: ["v*"]
+permissions:
+  contents: write  # to upload the binaries to the release
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: ["linux"]
+        arch: ["amd64", "arm64"]
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v4
+        with:
+          go-version: "1.20"
+      - run: mkdir build
+      - run: go build -trimpath -o build/lk-jwt-service_${{ matrix.os }}_${{ matrix.arch }} ./main.go
+        env:
+          GOOS: ${{ matrix.os }}
+          GOARCH: ${{ matrix.arch }}
+      - name: "Upload binary as artifact"
+        uses: actions/upload-artifact@v3
+        with:
+          name: build
```

Comment:
I don't know if this being the same name for both architectures will be a problem. They might overwrite each other?